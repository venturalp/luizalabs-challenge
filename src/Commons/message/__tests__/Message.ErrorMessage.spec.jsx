import { render } from '@testing-library/react'
import { ThemeWrapper } from 'Commons/tests/Tests.Helpers'
import React from 'react'
import { ErrorMessage } from '../Message.ErrorMessage'

describe('test ErrorMessage', () => {
  it('test with prop align', () => {
    const { asFragment } = render(
      <ThemeWrapper>
        <ErrorMessage error="foo bar" align="left" />
      </ThemeWrapper>,
    )

    expect(asFragment()).toMatchSnapshot()
  })
})
