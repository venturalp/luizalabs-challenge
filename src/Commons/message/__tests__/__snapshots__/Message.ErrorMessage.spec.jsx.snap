// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test ErrorMessage test with prop align 1`] = `
<DocumentFragment>
  <p
    class="sc-bdfBwQ fzWcBw"
  >
    foo bar
  </p>
</DocumentFragment>
`;
